### PR TITLE
Structure and CustomUser model

### PR DESCRIPTION
**Pull request by TareqMonwer**
root .vscode is not removed while I added it on .gitignore

### CODE REVIEW

File: backend/core/admin.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from django.contrib import admin
```
Comment (tanvirstreame) [inline backend/core/admin.py]: remove forms.py

Comment (TareqMonwer) [reply backend/core/admin.py]: why?

Comment (TareqMonwer) [reply backend/core/admin.py]: It is required for usercreation and update in django admin.

File: backend/core/forms.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth import get_user_model
```
Comment (tanvirstreame) [inline backend/core/forms.py]: In rest framework there are no need of forms.
We will use serializer. It is alternative form

File: backend/core/models.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from django.db import models
```
Comment (tanvirstreame) [inline backend/core/models.py]: What is this?
Where are exta fields instead of pass?

File: backend/core/admin.py (hunk 1 above)
Comment (tanvirstreame) [reply backend/core/admin.py]: Make all these thing in model......
No need to declare here
admin.site.register(CustomUser, CustomUserAdmin)
 Will do all the work

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (tanvirstreame) [inline .vscode/settings.json]: do we need it?

File: backend/.vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (tanvirstreame) [inline backend/.vscode/settings.json]: do we need it?

File: backend/config/urls.py [hunk 6]
```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api/v1/', include('core.urls')),
```
Comment (tanvirstreame) [inline backend/config/urls.py]: if api/v1/ for core app only

then how can we use for same api/v1/ for other app?

File: .vscode/settings.json (hunk 4 above)
Comment (TareqMonwer) [reply .vscode/settings.json]: missed .gitignore.

File: backend/config/urls.py (hunk 6 above)
Comment (TareqMonwer) [reply backend/config/urls.py]: api/v1/ will be the root url. We can change them when we add new apps.

**Pull request merged**